packages/auto_mappr/lib/src/models/auto_mappr_config.dart

```diff
@@ -46,31 +46,44 @@ class AutoMapprConfig {
       '/// Available mappings:',
       for (final mapper in mappers) _getTypeMappingDocs(mapper),
       '/// {@endtemplate}',
-      ..._modulesDocComment(),
+      ..._importsDocComment(),
+      ..._delegatesDocComment(),
     ];
   }
 
-  List<String> _modulesDocComment() {
+  List<String> _importsDocComment() {
     return [
-      if (modulesList.isNotEmpty) ...[
+      if (importsList.isNotEmpty) ...[
```

Comment:
importsList should be named "includesList" to be concise

Reply:
Fixed.